pages/shared/groupWorkInfoContainer.ejs

```diff
@@ -0,0 +1,57 @@
+<div class="container-fluid">
+  <div class="row">
+    <% if (permissions.student_authz_join) { %>
+    <div class="col-sm bg-light py-4 px-4 border">
+      <p>This is a group assessment. Use your join code to invite others to join the group.</p>
+      <span class="badge badge-secondary">Group Name</span>
+      <span id="group-name"><%= groupinfo[0].name %></span><br>
+      <span class="badge badge-secondary">join Code</span>
+      <span id='join-code'><%= joincode %></span><br>
+    </div>
+    <% } %>
+    <div class="col-sm bg-light py-4 px-4 border">
+      <% if (permissions.student_authz_leave) { %>
+      <div style="display: flex; justify-content: flex-end">
+        <button type="button" class="btn btn-danger" data-toggle="modal" data-target="#leaveModal">
+          Leave the Group
+        </button>
+        <!-- Modal -->
+        <div class="modal fade" id="leaveModal" tabindex="-1" role="dialog" aria-labelledby="leaveModalLabel"
+          aria-hidden="true">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title" id="leaveModalLabel">Confirm leave group</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                <p>Are you sure you want to leave the group?</p>
+                <p>You will lose access to any work done by the group and you might not be able to re-join later.</p>
+              </div>
+              <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                <form name="leave-group-form" method="POST">
+                  <input type="hidden" name="__action" value="leaveGroup">
+                  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                  <button id="leave-group" type="submit" class="btn btn-danger">Leave group</button>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <% } %>
+      <span class="badge badge-secondary">Group Members</span>
```

Comment:
I also don't think a badge is quite the right thing here. Perhaps an `<h6>` or just bold text?